extensions/wikia/PortableInfobox/tests/PortableInfoboxDataServiceTest.php

```diff
@@ -127,41 +184,74 @@ public function testConstructor() {
 		$this->assertEquals( [ ], $result );
 	}
 
-	protected function getInfoboxData() {
-		return [ [ 'data' => [ [ "type" => "data",
-								 "data" => [
-									 "value" => "AAAA",
-									 "label" => "BBBB"
-								 ]
-							   ], [ "type" => "image",
-									"data" => [ [
-										"key" => "Test.jpg",
-										"alt" => null,
-										"caption" => null,
-									] ]
-							   ], [ "type" => "image",
-									"data" => [ [
-										"key" => "Test2.jpg",
-										"alt" => null,
-										"caption" => null
-									] ] ] ] ],
-				 [ 'data' => [ [ "type" => "image",
-								 "data" => [ [
-									 "key" => "Test2.jpg",
-									 "alt" => null,
-									 "caption" => null
-								 ] ] ] ] ] ];
+	protected function getInfoboxPageProps() {
+		return [
+			[
+				'parser_tag_version' => PortableInfoboxParserTagController::PARSER_TAG_VERSION,
+				'data' => [
+					[
+						'type' => 'data',
+						'data' => [
+							'value' => 'AAAA',
+							'label' => 'BBBB'
+						]
+					],
+					[
+						'type' => 'image',
+						'data' => [
+							[
+								'key' => 'Test.jpg',
+								'alt' => null,
+								'caption' => null,
+							]
+						]
+					],
+					[
+						'type' => 'image',
+						'data' => [
+							[
+								'key' => 'Test2.jpg',
+								'alt' => null,
+								'caption' => null
+							]
+						]
+					]
+				],
+				'metadata' => []
+			],
+			[
+				'parser_tag_version' => PortableInfoboxParserTagController::PARSER_TAG_VERSION,
+				'data' => [
+					[
+						'type' => 'image',
+						'data' => [
+							[
+								'key' => 'Test2.jpg',
+								'alt' => null,
+								'caption' => null
+							]
+						]
+					]
+				],
+				'metadata' => []
+			]
+		];
 	}
 }
 
-class TemplateHelperDummy {
+class ParsingHelperDummy {
 
-	public function __construct( $hidden = false ) {
-		$this->hidden = $hidden;
+	public function __construct( $infoboxesData = null, $includeonlyInfoboxesData = null ) {
+		$this->infoboxesData = $infoboxesData;
+		$this->includeonlyInfoboxesData = $includeonlyInfoboxesData;
 	}
 
-	public function parseInfoboxes( $title ) {
-		return $this->hidden;
+	public function parseIncludeonlyInfoboxes( $title ) {
+		return $this->includeonlyInfoboxesData;
+	}
+
+	public function reparseArticle( $title ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$title". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)